jwst/lib/set_telescope_pointing.py

```diff
@@ -8,16 +8,15 @@
 reference pixel of a desired aperture is calculated.
 
 The transformations are defined by the Technical Reference JWST-STScI-003222,
-SM-12. This document has undergone a number of revisions. For JWST release
-v1.3.1, the version implemented is based on an internal email version produced
-2021-07.
+SM-12. This document has undergone a number of revisions. The current version
+implemented is based on an internal email version Rev. C, produced 2021-11.
```

Comment:
Jonathon, can you forward me a copy of the new version of the TR?

Reply:
Sent in email.